sections/exchange/hooks/useExchange.tsx

```diff
@@ -661,12 +672,12 @@ const useExchange = ({
 					const gasPriceWei = gasPriceInWei(gasPrice);
 
 					const tx = await contract.approve(oneInchApproveAddress, ethers.constants.MaxUint256, {
-						gasLimit: normalizeGasLimit(Number(gasEstimate)),
+						gasLimit: isL2 ? Number(gasEstimate) : normalizeGasLimit(Number(gasEstimate)),
```

Comment:
maybe better to do `normalizeGasLimit(gasEstimate, isL2)`

also use `wei` instead of `Number` here reccomended (gwei is done with `wei(gasPrice, 9)`)

Reply:
I'd like to be explicit with the ternary, especially because there's no "normalization" being done to the L2 gas limit. 

But honestly, I don't even know why it's called normalize. All we do is add a buffer on L1.